### DISCUSSION TITLE
Is support for Amstrad CPC 6128 possible?

### DISCUSSION THREAD

**Discussion by ocalvo** [EtchedPixels/FUZIX#1081] [Q&A]
I see that Amstrad NC200 is supported. I wonder what it will take to support the CPC 6128?

Thanks,

Oscar

**Comment by EtchedPixels** [comment]
ZX Spectrum +3 and the CPC6128 are pretty close in terms of memory management and the like. 6128 has rather different and better video, and some differences in how the floppy setup works but I think I'd start from the +3 if I was going to tackle it.
Once you get more than 128K RAM it's a lot more interesting as whilst the memory mappings are a bit weird you could build a CPC6xx target that made proper use of the 512K say, whereas the equivalent on the speccy is generally 512K hidden in 8 or 16K windows in the low 16K and not accessible directly from +3 mode.
